packages/augur-ui/src/modules/auth/components/connect-account/connect-account.tsx

```diff
@@ -9,6 +9,8 @@ import { LoginAccount } from 'modules/types';
 
 import Styles from 'modules/auth/components/connect-account/connect-account.styles.less';
 import ToggleHeightStyles from 'utils/toggle-height.styles.less';
+import { LoginAccount } from 'modules/types';
+import { loadUniverseDetails } from 'modules/universe/actions/load-universe-details';
```

Comment:
not needed here. `loadUniverseDetails` should be called in the universe selector modal.